src/utilities/utilities.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { getButtonClasses, getColorSettings } from "./utilities";
+
+describe("utilities", () => {
```

Comment:
Is this a spec file because its testing UI elements?

Reply:
This is an Angular holdover. When you run `ng generate` in an Angular project, it will automatically create a test file in with `spec.ts`.